test/Microsoft.AspNetCore.Mvc.FunctionalTests/FiltersTest.cs

```diff
@@ -554,5 +554,53 @@ public FiltersTest(MvcTestFixture<FiltersWebSite.Startup> fixture)
                 $"CurrentCulture:{expected},CurrentUICulture:{expected}",
                 await response.Content.ReadAsStringAsync());
         }
+
+        [Fact]
+        public async Task RequestSizeLimit_ThrowsForInvalidSize()
```

Comment:
It actually might be the case that we have to do unit tests. The test server may not implement this.